django_multitenant/mixins.py

```diff
@@ -44,25 +44,28 @@ def __init__(self, *args, **kwargs):
         super(TenantModelMixin, self).__init__(*args, **kwargs)
 
     def _do_update(self, base_qs, using, pk_val, values, update_fields, forced_update):
-        #adding tenant filters for save
-        #Citus requires tenant_id filters for update, hence doing this below change.
+        # adding tenant filters for save
+        # Citus requires tenant_id filters for update, hence doing this below change.
 
+        model = base_qs.model
```

Comment:
sorry if this is a stupid question... why model is required here? i don't see where it's used in the function... does self._meta.model the same as the model here?

Reply:
Not a bad question, just bad code.